inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/flink/IntergrationTaskRunner.java

```diff
@@ -172,6 +172,12 @@ public void run() {
                         flinkInfo.setSavepointPath(savepointPath);
                         log.info("the jobId :{} status: {} ", flinkInfo.getJobId(),status);
                     }
+                    JobStatus jobStatus = flinkService.getJobStatus(flinkInfo.getJobId());
+                    if (jobStatus.isTerminalState()) {
+                        log.info("stop  job : {} status : {} ,success in backend", flinkInfo.getJobId(),jobStatus);
```

Comment:
Change the log info to `log.info("stop job {}, status: {}, success in backend", flinkInfo.getJobId(), jobStatus);`

Reply:
done